sdk/formrecognizer/azure-ai-formrecognizer/swagger/README.md

```diff
@@ -0,0 +1,33 @@
+# Azure Cognitive Service - Form Recognizer for Java
+
+> see https://aka.ms/autorest
+
+### Setup
+```ps
+Fork and clone https://github.com/Azure/autorest.java 
+git checkout v4
+git submodule update --init --recursive
+npm install
+npm install -g autorest
+```
+
+### Generation
+```ps
+cd <swagger-folder>
+autorest --java --use=C:/work/autorest.java
+```
+
+### Code generation settings
+``` yaml
+input-file: https://raw.githubusercontent.com/kristapratico/azure-rest-api-specs/form-recognizer-patch/specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json
```

Comment:
Is this the custom Swagger that is being used while the service team is working on one? If this isn't customized should point to the standard Swagger.

Reply:
yes, the correct swagger is being still worked on by the service team. This is the correct one until then.